composer.json

```diff
@@ -18,7 +18,7 @@
         "source": "https://github.com/cakephp/cakephp"
     },
     "require": {
-        "php": ">=5.5.9",
+        "php": ">=5.6.0",
```

Comment:
@ADmad appveyor should be happy now. I didn't remember but the drivers for 5.6 were already in the zip file